### PR TITLE
PT_BR Translations

### PR DESCRIPTION
**Pull request by rafaelbsd**
Updates to mailscanner/langagues/pt_br.php

### CODE REVIEW

File: mailscanner/languages/pt_br.php [hunk 1]
```diff
@@ -30,44 +30,44 @@
...
+    'badcontentinfected03' => 'Conteúdo Bloq./Infectado',
     'whitelisted03' => 'Lista Branca',
     'blacklisted03' => 'Lista Negra',
-    'notverified03' => 'N&atilde;o Verificado',
-    'mailscanner03' => 'MailScanner:',
-    'none03' => 'None',
-    'yes03' => 'YES',
-    'no03' => 'NO',
+    'notverified03' => 'Não Verificado',
+    'mailscanner03' => 'Mailscanner:',
```
Comment (endelwar) [inline mailscanner/languages/pt_br.php]: Please leave MailScanner name as is

File: mailscanner/languages/pt_br.php [hunk 2]
```diff
@@ -442,273 +442,273 @@
...
-    'errorfilterdomainforbidden12' => 'Error: You don\'t have the permissions, to change filters of users of domain %s',
-    'errorreportnodomainforbidden12' => 'Error: You don\'t have the permissions, to send spam reports for users without domain',
-    'errorreportdomainforbidden12' => 'Error: You don\'t have the permissions, to send spam reports for users of domain %s',
-    'noReportsEnabled12' => 'Error: You have to enable daily reports to request one',
+    'auditlog0112' => 'Novo',
+    'auditlog0212' => 'criado',
+    'auditlog0312' => 'Alteração de tipo de usuário para',
+    'auditlogfrom12' => 'de',
+    'auditlogto12' => 'para',
+    'auditlog0412' => 'Usuário %s excluído	',
```
Comment (endelwar) [inline mailscanner/languages/pt_br.php]: Are these spaces necessary?

**Review by endelwar — CHANGES_REQUESTED** [review]

File: mailscanner/languages/pt_br.php (hunk 2 above)
Comment (rafaelbsd) [reply mailscanner/languages/pt_br.php]: Hi... no, they are not. I'll fix it.

**Comment by rafaelbsd** [comment]
Already fixed those unecessary spaces.

File: mailscanner/languages/pt_br.php (hunk 1 above)
Comment (rafaelbsd) [reply mailscanner/languages/pt_br.php]: OK. Fixed it.

File: mailscanner/languages/pt_br.php (hunk 2 above)
Comment (rafaelbsd) [reply mailscanner/languages/pt_br.php]: Fixed it.

**Review by rafaelbsd — COMMENTED** [review]
Fixed MailScanner name and unnecessary spaces.

**Review by rafaelbsd — COMMENTED** [review]
MailScanner name

**Review by endelwar — APPROVED** [review]

**Pull request merged**